Modules/posixmodule.c

```diff
@@ -5224,9 +5225,17 @@ parse_file_actions(PyObject *file_actions,
                 {
                     goto fail;
                 }
+                int error = PyList_Append(temp_buffer, path);
+                if (error) {
```

Comment:
Seems the `error` variable is used only here. In this case it would be better to inline it.

Reply:
Done in commit 80918db